.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
 typings
-dist
```

Comment:
Since typescript projects have to be transpiled anyway, you can leave it ignored. The dist-folder only was useful for angular 1 projects.